From: Scott Kitterman <scott@kitterman.com>
Date: Mon, 20 Apr 2020 10:59:54 -0700
Subject: Include pkg-resources in freeze system wheels

Bug-Debian: https://bugs.debian.org/871790
Origin: vendor
Forwarded: not-needed
Last-Update: 2020-04-04

  * Add pkg-resources to pip freeze 'DEV_PKGS' so this Debian unique system
    wheel does not show up when pip freeze is run (Closes: #871790)
---
 src/pip/_internal/commands/freeze.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/pip/_internal/commands/freeze.py b/src/pip/_internal/commands/freeze.py
index f8de335..8eb1cd0 100644
--- a/src/pip/_internal/commands/freeze.py
+++ b/src/pip/_internal/commands/freeze.py
@@ -18,6 +18,7 @@ def _dev_pkgs() -> AbstractSet[str]:
 
     if _should_suppress_build_backends():
         pkgs |= {"setuptools", "distribute", "wheel"}
+        pkgs |= {"setuptools", "distribute", "wheel", "pkg-resources"}
 
     return pkgs
 
